cms/envs/devstack.py

```diff
@@ -118,3 +118,8 @@ def should_show_debug_toolbar(_):
 
 # Dummy secret key for dev
 SECRET_KEY = '85920908f28904ed733fe576320db18cabd7b6cd'
+
+########################## Third Party Auth #######################
+
+if FEATURES.get('ENABLE_THIRD_PARTY_AUTH') and 'third_party_auth.dummy.DummyBackend' not in AUTHENTICATION_BACKENDS:
+    AUTHENTICATION_BACKENDS = ['third_party_auth.dummy.DummyBackend'] + list(AUTHENTICATION_BACKENDS)
```

Comment:
I really don't like copying and pasting settings across lms/cms. I understand that we currently have no shared common settings file in which to place this stuff, so we may have to live with this if configuration models don't work out.